cmd/root.go

```diff
@@ -24,14 +29,26 @@ func Execute() {
 	}
 }
 
-var identifier string
-
 func init() {
-	// Here you will define your flags and configuration settings.
-	// Cobra supports persistent flags, which, if defined here,
-	// will be global for your application.
+	smParams := model.ShardingManagerParams{}
+
+	rootCmd.PersistentFlags().StringVar(&smParams.KubeconfigPath, "kube_config", "", "Use a Kubernetes configuration file instead of in-cluster configuration")
+	//defines the identity of sharding manager instance - logical name for group of resources handled by an instance of sharding manager. This is used to initialize configuration for resources to be handled
+	rootCmd.Flags().StringVar(&smParams.ShardingManagerIdentity, "shard-identity", "devx", "Identity of the sharding manager instance")
```

Comment:
We can add a description, saying that the value assigned to this variable, will be used as a label, with a key of `admiral.io/shardingMangerIdentity`